frontend/src/metabase/dashboard/reducers.js

```diff
@@ -338,7 +338,11 @@ const parameterValues = handleActions(
 
 const draftParameterValues = handleActions(
   {
-    [INITIALIZE]: { next: () => ({}) },
+    [INITIALIZE]: {
```

Comment:
here we apply common logic on resetting state only when `clearCache` is passed. otherwise it's navigatingBack